pkg/corerp/renderers/container/render.go

```diff
@@ -109,6 +109,20 @@ func (r Renderer) GetDependencyIDs(ctx context.Context, dm v1.DataModelInterface
 		}
 	}
 
+	for _, envvars := range properties.Container.Env {
+		if envvars.ValueFrom != nil && envvars.ValueFrom.SecretRef != nil {
+			if strings.HasPrefix(envvars.ValueFrom.SecretRef.Source, "/") {
```

Comment:
Does this mean that if the **Source** has the prefix "/" it could be a resource id? If there is a secret source that doesn't have this prefix, should we log this or error out?

Reply:
Source is either the resource id of a radius resource or a kubernetes secret reference.